recipes/_user.rb

```diff
@@ -73,6 +73,7 @@
   members node['omnibus']['build_user']
   append true
   action :modify
+  not_if { windows? } # Cannot modify a group that doesn't exist
```

Comment:
Wouldn't it make more sense to use `ignore_failure true` here also? If the group represented by `node['omnibus']['build_user_group']` was created above (or exists) we want to at least attempt to add `node['omnibus']['build_user']` to said group.